File: Data Structures & Algorithms/count-paths/submission-0.cpp
class Solution {
public:
    int uniquePaths(int m, int n) {
        vector<vector<int>> dp(m+1,vector<int>(n+1,0));
        dp[m-1][n-1] = 1;
        for(int r = m-1; r >=0; r--){
            for(int c = n-1; c >= 0; c--){
                dp[r][c] += dp[r+1][c] + dp[r][c+1];
            }
        }
        return dp[0][0];   
    }
};

File: Data Structures & Algorithms/level-order-traversal-of-binary-tree/submission-0.py
# Definition for a binary tree node.
# class TreeNode:
#     def __init__(self, val=0, left=None, right=None):
#         self.val = val
#         self.left = left
#         self.right = right

class Solution:
    def levelOrder(self, root: Optional[TreeNode]) -> List[List[int]]:
        output = []
        temp_list = []
        if root is None:
            return output
        queue_nodes = deque([root])

        while queue_nodes:
            temp_list = []
            for i in range(len(queue_nodes)):
                node = queue_nodes.popleft()
                temp_list.append(node.val)
                if node.left:
                    queue_nodes.append(node.left)
                if node.right:
                    queue_nodes.append(node.right)
            output.append(temp_list)
        
        return output
        